### PR TITLE
Add KSlider with supporting Actions, Assertions and Matcher.

### PR DESCRIPTION
**Pull request by carlochum**

### CODE REVIEW

File: kakao/src/main/kotlin/com/agoda/kakao/common/matchers/SliderMatcher.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * @param value of value that matched the view which is Slider
+ */
+class SliderMatcher(private val value: Float) : BoundedMatcher<View, Slider>(Slider::class.java) {
+    override fun matchesSafely(view: Slider?) = view?.let { it.value == value } ?: false
+
+    override fun describeTo(description: Description) {
+        description.appendText("slider value is: $value")
+    }
+}
+
```
Comment (Vacxe) [inline kakao/src/main/kotlin/com/agoda/kakao/common/matchers/SliderMatcher.kt:22]: Empty line

**Review by Vacxe — COMMENTED** [review]
Please provide an example for new classes in sample app

File: sample/src/androidTest/kotlin/com/agoda/sample/screen/SliderScreen.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.agoda.sample.screen
+
+import com.agoda.kakao.screen.Screen
+import com.agoda.kakao.slider.KSlider
+import com.agoda.kakao.text.KButton
+import com.agoda.sample.R
+
+class SliderScreen : Screen<SliderScreen>() {
+    val slider = KSlider { withId(R.id.slider) }
+    val buttonValue50 = KButton { withId(R.id.button_value_5) }
```
Comment (Vacxe) [inline sample/src/androidTest/kotlin/com/agoda/sample/screen/SliderScreen.kt:10]: R.id.button_value_**5** assigned for buttonValue**50**. Why?

Comment (carlochum) [reply sample/src/androidTest/kotlin/com/agoda/sample/screen/SliderScreen.kt:10]: ahh, it's meant to be buttonValue5, used to be 50 but i changed it to use smaller numbers.

**Review by Vacxe — APPROVED** [review]
@carlochum please accept a CLA (You can click on 'details' on bottom checks for cla)

**Review by Unlimity — APPROVED** [review]

**Pull request merged**